### ISSUE TITLE
bower.json

### ISSUE THREAD

**Issue by Proghat** [chaijs/chai-jquery#50]
I would like to download chai-jquery using bower but I don't see `bower.json` file in repository. Still I can find (and download) it on bower.io: http://bower.io/search/?q=chai-jquery.

How it's possible? Can we add `bower.json` to repository?

I ask because I want to use chai-jquery in my rails project through [Rails Assets](https://rails-assets.org/) and it requires `bower.json`:
> You can now require BOWER_PACKAGE_NAME in your stylesheets and JavaScript files—all assets exposed as main in bower.json will be automatically included.

**Comment by jaredmoody** [comment]
:+1:

**Comment by T1st3** [comment]
At the moment, bower.io can not find chai-jquery anymore (using the provided link).

However, it is still possible to get chai-jquery with bower, and you have 2 ways to do it:

directly on the CLI:

```
bower install chai-jquery
```


or by inclusion in your bower.json's dependencies or devDependencies:

```
  "devDependencies": {
    "chai": "~3.1",
    "chai-jquery": "~2.0"
  }
```

**Comment by keithamus** [comment]
@Proghat thanks for filing the issue.

@Proghat @jaredmoody @T1st3 if you'd like, any one of you can work on a PR to add the bower.json back in, and I'll merge it :smile:.

**Comment by T1st3** [comment]
just created a PR for it (#70)

**Comment by T1st3** [comment]
@keithamus: this one should be closed (ref: #70)
To make it clear for everybody, bower.json was created in version 2.01 of `chai-jquery`. Please update if you want this feature!

**Comment by keithamus** [comment]
Thanks @T1st3 :smile:
